keylime/ima/ima.py

```diff
@@ -610,6 +615,30 @@ def canonicalize_allowlist(alist_bytes, checksum=""):
     return alist
 
 
+def ima_policy_db_contents(ima_policy_name: str, ima_policy: str, tpm_policy: bool = False) -> dict:
+    "Assembles an ima policy dictionary to be written on the database"
```

Comment:
Function comments have to start and end with 3 `"""`.

Reply:
oops